{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://github.com/pytest-dev/pytest-faulthandler",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pytest-faulthandler/pytest-faulthandler-1.3.0.zip",
    "subdirs": [
      "win-64"
    ],
    "summary": "py.test plugin that activates the fault handler module for tests",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "1.3.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main"
  ],
  "package": "pytest-faulthandler-1.3.0-py35_0.tar.bz2",
  "repodata": {
    "build": "py35_0",
    "build_number": 0,
    "depends": [
      "pytest",
      "python 3.5*"
    ],
    "license": "MIT",
    "md5": "a846d982a0fdbffed88ae8f283e57e80",
    "name": "pytest-faulthandler",
    "sha256": "12d666f3248ba11084a2f4ca7c9e5aeba9ac5b7e1fba7e983e4ffe13b7a17140",
    "size": 4292,
    "subdir": "win-64",
    "version": "1.3.0"
  },
  "repodata_version": 1,
  "subdir": "win-64",
  "url": "https://conda.anaconda.org/conda-forge/win-64/pytest-faulthandler-1.3.0-py35_0.tar.bz2"
}